recipes/wpewebkit/0008-Remove-absolute-path-from-libWPEInjectedBundle.so-on.patch

```diff
@@ -0,0 +1,27 @@
+From 26b5a5035953a8fcfa93c37342596f79ee7d2509 Mon Sep 17 00:00:00 2001
+From: Jani Hautakangas <jani@igalia.com>
+Date: Thu, 24 Mar 2022 09:41:17 +0200
+Subject: [PATCH] Remove absolute path from libWPEInjectedBundle.so on Android
+
+Android application native library dir doesn't support subdirectories
+---
+ Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp | 3 +--
+ 1 file changed, 1 insertion(+), 2 deletions(-)
+
+diff --git a/Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp b/Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp
+index 990e8d04..7178e313 100644
+--- a/Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp
++++ b/Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp
+@@ -323,8 +323,7 @@ static const char* injectedBundleDirectory()
+         G_DIR_SEPARATOR_S "injected-bundle" G_DIR_SEPARATOR_S;
+     return injectedBundlePath;
+ #elif PLATFORM(WPE)
+-    static const char* injectedBundlePath = PKGLIBDIR G_DIR_SEPARATOR_S "injected-bundle" G_DIR_SEPARATOR_S;
+-    return injectedBundlePath;
++    return "";
```

Comment:
If we were to try to submit the change upstream at some point it would need guards (something like `#ifdef __ANDROID__`) but this will do for now.

Reply:
__ANDROID__ is not getting defined currently

Reply:
You are right, it's `ANDROID` and `__ANDROID_API__`:

```
cerbero % grep ANDROID config/android.config      
env['ANDROID_NDK_HOME'] = toolchain_prefix
defines = '-DANDROID -DPIC -D__ANDROID_API__=%s ' % (v)
cerbero %